packages/atlas/src/views/viewer/VideoView/MoreVideos.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+
+import { useBasicVideos } from '@/api/hooks'
+import { CategoryLink } from '@/components/CategoryLink'
+import { GridItem, LayoutGrid } from '@/components/LayoutGrid'
+import { Button } from '@/components/_buttons/Button'
+import { ChannelLink } from '@/components/_channel/ChannelLink'
+import { SvgActionChevronR } from '@/components/_icons'
+import { VideoTileViewer } from '@/components/_video/VideoTileViewer'
+import { absoluteRoutes } from '@/config/routes'
+import { useMediaMatch } from '@/hooks/useMediaMatch'
+
+import { MoreFrom, MoreVideosContainer, SeeMoreButton } from './VideoView.styles'
+
+type MoreVideosProps = {
+  channelId?: string
+  channelName?: string | null
+  categoryId?: string
+  categoryName?: string | null
+  type: 'channel' | 'category'
+}
+
+const NUMBER_OF_VIDEOS = 6
+
+export const MoreVideos: React.FC<MoreVideosProps> = ({ channelId, channelName, categoryId, categoryName, type }) => {
+  const where = type === 'channel' ? { channel: { id_eq: channelId } } : { category: { id_eq: categoryId } }
+  const { videos = [], loading } = useBasicVideos({ where, limit: NUMBER_OF_VIDEOS })
```

Comment:
I think we may want to include sorting here (or in the query directly) to make sure we get the latest videos